### PR TITLE
Add pytorch-1.12.0-aarch64 runtime image

### PR DESCRIPTION
**Pull request by Guo-Peilin**

### CODE REVIEW

File: .github/workflows/pytorch112_aarch64.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      uses: actions/checkout@v2.4.0
+    - name: Build Dev Docker
+      run: |
+        set -e
+        git submodule sync
+        git submodule update --depth=1 --init --recursive
+        cp /etc/apt/sources.list .
+        docker build -t disc-dev-cpu-aarch64 \
+          --build-arg BASEIMAGE=ubuntu:20.04 \
+          --build-arg DISC_HOST_TF_VERSION="tensorflow-aarch64==2.8" \
```
Comment (qiuxiafei) [inline .github/workflows/pytorch112_aarch64.yml:27]: why do we need TF in pytorch image?

File: .github/workflows/pytorch112_aarch64.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+        GITHUB_PULL_REQUEST: ${{ github.event.number }}
+        LOCAL_DEV_DOCKER: disc-dev-cpu-aarch64
+        REMOTE_DEV_DOCKER: bladedisc:latest-devel-cpu-aarch64
+      run: |
+        set -e
+        echo "Try to deploy runtime docker image..."
+        source $HOME/.cache/proxy_config
+        bash ./scripts/ci/deploy_tf_wrapper.sh
```
Comment (qiuxiafei) [inline .github/workflows/pytorch112_aarch64.yml:57]: `deploy_tf_wrapper.sh` ?

File: .github/workflows/pytorch112_aarch64.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+        GITHUB_PULL_REQUEST: ${{ github.event.number }}
+        RUNTIME_BASEIMAGE: bladedisc/bladedisc:latest-devel-cpu-aarch64
+        RUNTIME_DOCKER_FILE: docker/runtime/Dockerfile.pytorch.aarch64
+        REMOTE_RUNTIME_DOCKER: bladedisc:latest-runtime-torch1.12.0-cpu-aarch64
+      run: |
+        set -e
+        echo "Try to deploy runtime docker image..."
+        source $HOME/.cache/proxy_config
+        bash ./scripts/ci/deploy_tf_wrapper.sh
```
Comment (qiuxiafei) [inline .github/workflows/pytorch112_aarch64.yml:73]: ditto

File: .github/workflows/pytorch112_aarch64.yml (hunk 1 above)
Comment (Guo-Peilin) [reply .github/workflows/pytorch112_aarch64.yml:27]: `pytorch112_aarch64.yaml` is mainly copy from `pytorch110_aarch64.yaml` except for the version number.
I will check if tf is necessary

File: .github/workflows/pytorch112_aarch64.yml (hunk 2 above)
Comment (Guo-Peilin) [reply .github/workflows/pytorch112_aarch64.yml:57]: https://github.com/alibaba/BladeDISC/pull/474/files
From above commit, it seems that we use `deploy_tf_wrapper.sh` both in `pytorch` and `tensorflow` at the very beginning. Also, pytorch110_aarch64.yaml use deploy_tf_wrapper.sh.

File: .github/workflows/pytorch112_aarch64.yml (hunk 1 above)
Comment (Guo-Peilin) [reply .github/workflows/pytorch112_aarch64.yml]: No need for tensorflow version

**Review by qiuxiafei — APPROVED** [review]
LGTM

**Pull request merged**